### PR TITLE
tools/tidb-lightning: update Lightning-related docs

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 

1. Updated download links to 3.0.2 and 2.1.14.
2. Documented the web interface for master version.
3. Documented the Grafana dashboard.
4. Reorganized the "deployment" page --- the full configuration is now moved into a dedicated "configuration" page, while the "deployment" only contains the essential configuration to simplify the steps.
5. The new "configuration" page also describes the CLI parameters of the associated tools.

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs/pull/1455
https://github.com/pingcap/docs/pull/1715/ (update importer sample config)

### Which version does your change affect? 

dev, v2.1, v3.0 and v3.1: `TOC.md`, `config.md`(new), `deployment.md`, `monitor.md`, `overview.md`, `web.md`(new), ~`download.md`~(updated in https://github.com/pingcap/docs-cn/pull/2050)

### CODE REVIEW

File: dev/reference/tools/tidb-lightning/config.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+## 命令行参数
+
+### `tidb-lightning`
+
+使用 `tidb-lightning` 可以对下列参数进行配置：
+
+| 参数 | 解释 | 对应配置 |
+|:----|:----|:----|
+| --config *file* | 从 *file* 读取全局设置。如果没有指定则使用默认设置。R | |
```
Comment (kennytm) [inline dev/reference/tools/tidb-lightning/config.md]:
```suggestion
| --config *file* | 从 *file* 读取全局设置。如果没有指定则使用默认设置。 | |
```

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 2]
```diff
@@ -55,9 +55,117 @@ scrape_configs:
...
+| Delivery duration | SST delivery | 将单个 SST 文件上传到 TiKV 集群所消耗的时间 |
+| SST process duration | Split SST | 将键值对流切分成若干 SST 文件所消耗的时间 |
+| SST process duration | SST upload | 上传单个 SST 文件所消耗的时间 |
+| SST process duration | SST ingest | Ingest 单个 SST 文件所消耗的时间 |
+| SST process duration | SST size | 单个 SST 文件的大小 |
+
 ## 监控指标
 
-本节将详细描述 `tikv-importer` 和 `tidb-lightning` 的监控指标。
+本节中将详细描述 Grafana 仪表盘支持的其他 `tikv-importer` 和 `tidb-lightning` 的监控指标。
```
Comment (kennytm) [inline dev/reference/tools/tidb-lightning/monitor.md]: This section is not directly related to Grafana (the frontend) anymore. These are metrics for Prometheus (the backend)

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 3]
```diff
@@ -55,6 +55,114 @@ scrape_configs:
...
+![第五行读取速度面板](/media/lightning-grafana-row-5.png)
+
+第五行面板显示读取速度，具体为：
+
+| 面板名称 | 功能 | 描述 |
+|:-----|:-----|:-----|
+| Chunk parser read block duration | read block | 读取一个字节块以准备解析所消耗的时间 |
+| Chunk parser read block duration | apply worker | 等待 `io-concurrency` 空闲所消耗的时间 |
+| SQL process duration | row encode | 解析和编码单行所消耗的时间 |
+| SQL process duration | block deliver | 将一组键/值对发送到 TiKV Importer 所消耗的时间 |
```
Comment (kennytm) [inline dev/reference/tools/tidb-lightning/monitor.md]: Please pick a single translation to the term "KV pair (key-value pair)"... I've seen "键/值对" here...

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 4]
```diff
@@ -55,6 +55,114 @@ scrape_configs:
...
+
+![第六行存储空间面板](/media/lightning-grafana-row-6.png)
+
+第六行面板显示存储空间使用情况，具体为：
+
+| 面板名称 | 功能 | 描述 |
+|:-----|:-----|:-----|
+| SQL process rate | data deliver rate | 向 TiKV Importer 发送键值对 data 的速度 |
+| SQL process rate | index deliver rate | 向 TiKV Importer 发送键值对 index 的速度 |
+| SQL process rate | total deliver rate | 键值对上传的总速度 |
```
Comment (kennytm) [inline dev/reference/tools/tidb-lightning/monitor.md]: ... but "键值对" here ...

File: dev/reference/tools/tidb-lightning/overview.md [hunk 5]
```diff
@@ -31,8 +31,10 @@ TiDB Lightning 整体工作原理如下：
 
 5. 当一个引擎文件数据写入完毕时，`tikv-importer` 便开始对目标 TiKV 集群数据进行分裂和调度，然后导入数据到 TiKV 集群。
 
-   引擎文件包含两种：**数据引擎**与**索引引擎**，各自又对应两种 KV 对：行数据和次级索引。通常行数据在数据源里是完全有序的，而次级索引是无序的。因此，数据引擎文件在对应区块写入完成后会被立即上传，而所有的索引引擎文件只有在整张表所有区块编码完成后才会执行导入。
+    引擎文件包含两种：**数据引擎**与**索引引擎**，各自又对应两种 KV 对：行数据和次级索引。通常行数据在数据源里是完全有序的，而次级索引是无序的。因此，数据引擎文件在对应区块写入完成后会被立即上传，而所有的索引引擎文件只有在整张表所有区块编码完成后才会执行导入。
```
Comment (kennytm) [inline dev/reference/tools/tidb-lightning/overview.md]: ... but "KV 对" here.

File: dev/reference/tools/tidb-lightning/overview.md [hunk 6]
```diff
@@ -31,8 +31,10 @@ TiDB Lightning 整体工作原理如下：
...
 
-6. 整张表相关联的所有引擎文件完成导入后，`tidb-lightning` 会对比本地数据源及下游集群的校验和 (checksum)，确保导入的数据无损，以及让 TiDB 分析 (`ANALYZE`) 这些新增的数据，以优化日后的操作。
+6. 整张表相关联的所有引擎文件完成导入后，`tidb-lightning` 会对比本地数据源及下游集群的校验和 (checksum)，确保导入的数据无损，然后让 TiDB 分析 (`ANALYZE`) 这些新增的数据，以优化日后的操作。同时，`tidb-lightning` 调整 `AUTO_INCREMENT` 值防止之后新增数据时发生冲突。
+
+    表的自增 ID 是通过估计行数的 **上届**得到的，与表的数据文件总大小成正比。因此，最后的自增 ID 通常比实际行数大得多。这属于正常现象，因为在 TiDB 中自增 ID [不一定是连续分配](/dev/reference/mysql-compatibility.md#auto-increment-id)。
```
Comment (kennytm) [inline dev/reference/tools/tidb-lightning/overview.md]: 上届 → 上界/上限 ?

File: dev/reference/tools/tidb-lightning/monitor.md (hunk 3 above)
Comment (anotherrachel) [reply dev/reference/tools/tidb-lightning/monitor.md]: Thanks for pointing it out. I translated the term “KV pair” into “键/值对” at first and then switched to "键值对“. I must have missed one here. I'll unify the translation in this PR, and open another PR to unify the translation across the docs. Thanks again.

**Comment by TomShawn** [comment]
@anotherrachel When the content is ready, please also update v3.1 as well, thanks~

File: dev/reference/tools/tidb-lightning/deployment.md [hunk 7]
```diff
@@ -179,7 +179,7 @@ TiDB Lightning 可随 TiDB 集群一起用 [Ansible 部署](/dev/how-to/deploy/o
 
 #### 第 2 步：下载 TiDB Lightning 安装包
 
-通过以下链接获取 TiDB Lightning 安装包（需选择与集群相同的版本）：
+通过以下链接获取 TiDB Lightning 安装包：
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/deployment.md]:
```suggestion
通过以下链接获取 TiDB Lightning 安装包（需选择与 TiDB 集群相同的版本）：
```

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 8]
```diff
@@ -55,6 +55,100 @@ scrape_configs:
...
+如果上述项的持续时间过长，则表示 TiDB Lightning 使用的磁盘运转太慢或 I/O 太忙。
+
+### 第六行：存储空间面板
+
+![第六行存储空间面板](/media/lightning-grafana-row-6.png)
+
+| 面板名称 | 描述 |
+|:-----|:-----|
+| SQL process rate - data deliver rate | 向 TiKV Importer 发送键值对 data 的速度 |
+| SQL process rate - index deliver rate | 向 TiKV Importer 发送键值对 index 的速度 |
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/monitor.md]:
```suggestion
| SQL process rate - index deliver rate | 向 TiKV Importer 发送索引键值对的速度 |
```

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 9]
```diff
@@ -55,6 +55,100 @@ scrape_configs:
...
+| SQL process rate - total deliver rate | 键值对上传的总速度 |
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/monitor.md]:
```suggestion
| SQL process rate - total deliver rate | 发送数据键值对及索引键值对的速度之和 |
```

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 10]
```diff
@@ -55,6 +55,100 @@ scrape_configs:
...
+| Total bytes - parser read size | TiDB Lightning 正在读取的字节数 |
+| Total bytes - data deliver size | 已发送到 TiKV Importer 的键值对 data 的字节数 |
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/monitor.md]:
```suggestion
| Total bytes - data deliver size | 已发送到 TiKV Importer 的数据键值对的字节数 |
```

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 11]
```diff
@@ -55,6 +55,100 @@ scrape_configs:
...
+| Total bytes - index deliver size | 已发送到 TiKV Importer 的键值对 index 的字节数 |
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/monitor.md]:
```suggestion
| Total bytes - index deliver size | 已发送到 TiKV Importer 的索引键值对的字节数 |
```

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 12]
```diff
@@ -55,6 +55,100 @@ scrape_configs:
...
+| Total bytes - storage_size/3 | TiKV 集群的占用大小的三分之一（3 为默认的副本数量）|
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/monitor.md]:
```suggestion
| Total bytes - storage_size/3 | TiKV 集群占用的存储空间大小的 1/3（3 为默认的副本数量）|
```

File: dev/reference/tools/tidb-lightning/overview.md [hunk 13]
```diff
@@ -5,34 +5,36 @@ category: reference
...
 
 5. 当一个引擎文件数据写入完毕时，`tikv-importer` 便开始对目标 TiKV 集群数据进行分裂和调度，然后导入数据到 TiKV 集群。
 
-   引擎文件包含两种：**数据引擎**与**索引引擎**，各自又对应两种 KV 对：行数据和次级索引。通常行数据在数据源里是完全有序的，而次级索引是无序的。因此，数据引擎文件在对应区块写入完成后会被立即上传，而所有的索引引擎文件只有在整张表所有区块编码完成后才会执行导入。
+    引擎文件包含两种：**数据引擎**与**索引引擎**，各自又对应两种键值对：行数据和次级索引。通常行数据在数据源里是完全有序的，而次级索引是无序的。因此，数据引擎文件在对应区块写入完成后会被立即上传，而所有的索引引擎文件只有在整张表所有区块编码完成后才会执行导入。
 
-6. 整张表相关联的所有引擎文件完成导入后，`tidb-lightning` 会对比本地数据源及下游集群的校验和 (checksum)，确保导入的数据无损，以及让 TiDB 分析 (`ANALYZE`) 这些新增的数据，以优化日后的操作。
+6. 整张表相关联的所有引擎文件完成导入后，`tidb-lightning` 会对比本地数据源及下游集群的校验和 (checksum)，确保导入的数据无损，然后让 TiDB 分析 (`ANALYZE`) 这些新增的数据，以优化日后的操作。同时，`tidb-lightning` 调整 `AUTO_INCREMENT` 值防止之后新增数据时发生冲突。
+
+    表的自增 ID 是通过估计行数的**上界**得到的，与表的数据文件总大小成正比。因此，最后的自增 ID 通常比实际行数大得多。这属于正常现象，因为在 TiDB 中自增 ID [不一定连续分配](/dev/reference/mysql-compatibility.md#auto-increment-id)。
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/overview.md]:
```suggestion
    表的自增 ID 是通过行数的**上界**估计值得到的，与表的数据文件总大小成正比。因此，最后的自增 ID 通常比实际行数大得多。这属于正常现象，因为在 TiDB 中自增 ID [不一定是连续分配的](/dev/reference/mysql-compatibility.md#auto-increment-id)。
```

File: dev/reference/tools/tidb-lightning/web.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
+---
+title: TiDB Lightning Web 界面
+summary: 了解 TiDB Lightning 的服务器模式。
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/web.md]:
```suggestion
summary: 了解 TiDB Lightning 的服务器模式——通过 Web 界面来控制 TiDB Lightning。
```

File: dev/reference/tools/tidb-lightning/web.md [hunk 15]
```diff
@@ -0,0 +1,80 @@
...
+category: reference
+---
+
+# TiDB Lightning Web 界面
+
+TiDB Lightning 支持在网页上查看导入进度或执行一些简单任务管理，切换为**服务器模式**后即可启用。本文将介绍服务器模式对应的 Web 界面。
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/web.md]:
```suggestion
TiDB Lightning 支持在网页上查看导入进度或执行一些简单任务管理，这就是 TiDB Lightning 的**服务器模式**。本文将介绍服务器模式下的 Web 界面和一些常见操作。
```

File: dev/reference/tools/tidb-lightning/web.md [hunk 16]
```diff
@@ -0,0 +1,80 @@
...
+
+如需启用服务器模式，你可以通过命令行参数 `--server-mode` 来启动 `tidb-lightning` 。
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/web.md]:
```suggestion
启用服务器模式的方式有如下几种：

1. 在启动 `tidb-lightning` 时加上命令行参数 `--server-mode`。
```

File: dev/reference/tools/tidb-lightning/web.md [hunk 17]
```diff
@@ -0,0 +1,80 @@
...
+
+```sh
+./tidb-lightning --server-mode --status-addr :8289
+```
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/web.md]: Indent this block

File: dev/reference/tools/tidb-lightning/web.md [hunk 18]
```diff
@@ -0,0 +1,80 @@
...
+
+或者在配置文件中设置 `lightning.server-mode`。
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/web.md]:
```suggestion
2. 在配置文件中设置 `lightning.server-mode`。
```

File: dev/reference/tools/tidb-lightning/web.md [hunk 19]
```diff
@@ -0,0 +1,80 @@
...
+
+```toml
+[lightning]
+server-mode = true
+status-addr = ':8289'
+```
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/web.md]: Indent this block

File: dev/reference/tools/tidb-lightning/web.md [hunk 20]
```diff
@@ -0,0 +1,80 @@
...
+
+TiDB Lightning 启动后，请访问 `http://127.0.0.1:8289` 来管理程序（实际的 URL 取决于 `status-addr` 设置）。
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/web.md]:
```suggestion
TiDB Lightning 启动后，可以访问 `http://127.0.0.1:8289` 来管理程序（实际的 URL 取决于你的 `status-addr` 设置）。
```

File: dev/reference/tools/tidb-lightning/web.md [hunk 21]
```diff
@@ -0,0 +1,80 @@
...
+
+服务器模式下，TiDB Lightning 不会立即开始运行，而是通过用户在 web 页面提交（多个）**任务**来导入数据。
+
+## 首页
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/web.md]:
```suggestion
## TiDB Lightning Web 首页
```

File: dev/reference/tools/tidb-lightning/web.md [hunk 22]
```diff
@@ -0,0 +1,80 @@
...
+
+![首页](/media/lightning-web-frontpage.png)
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/web.md]:
```suggestion
![TiDB Lightning Web 首页](/media/lightning-web-frontpage.png)
```

File: dev/reference/tools/tidb-lightning/web.md [hunk 23]
```diff
@@ -0,0 +1,80 @@
...
+
+标题栏上图标所对应的功能，从左到右依次为：
+
+| 图标 | 功能 |
+|:----|:----|
+| "TiDB Lightning" | 返回首页 |
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/web.md]:
```suggestion
| "TiDB Lightning" | 点击即返回首页 |
```

File: dev/reference/tools/tidb-lightning/web.md [hunk 24]
```diff
@@ -0,0 +1,80 @@
...
+
+每个面板都包含用于描述表状态的卡片。
+
+## 提交任务
+
+点击标题栏的 **+** 图标提交任务。
+
+![提交任务对话框](/media/lightning-web-submit.png)
+
+任务 (task) 为 TOML 格式的文件，具体参考[TiDB Lightning 任务配置](/dev/reference/tools/tidb-lightning/config.md#tidb-lightning-任务配置参数)。你也可以点击 **UPLOAD** 上传一个本地的 TOML 文件。
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/web.md]:
```suggestion
任务 (task) 为 TOML 格式的文件，具体参考 [TiDB Lightning 任务配置](/dev/reference/tools/tidb-lightning/config.md#tidb-lightning-任务配置参数)。你也可以点击 **UPLOAD** 上传一个本地的 TOML 文件。
```

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 25]
```diff
@@ -55,6 +55,100 @@ scrape_configs:
...
+如果使用其他方式部署 TiDB Lightning，需先[导入仪表盘（JSON 文件）](https://raw.githubusercontent.com/pingcap/tidb-ansible/master/scripts/lightning.json)。
+
+### 第一行：速度面板
+
+![第一行速度面板](/media/lightning-grafana-row-1.png)
+
+| 面板名称 | 描述 |
+|:-----|:-----|
+| Import speed - write from lightning | 从 TiDB Lightning 向 TiKV Importer 发送键值对的速度，取决于每个表的复杂性 |
+| Import speed - upload to tikv | TiKV Importer 上传所有 TiKV 副本的总体速度 |
```
[the remark this answers is not in the document]
Comment (anotherrachel) [reply dev/reference/tools/tidb-lightning/monitor.md]: 确认了下~上传的是 SST 文件。

**Comment by anotherrachel** [comment]
@yikeke PTAL, thanks!
